CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [15-5.1] - 2024-11-20
+* The entire WebDAV URL is now shown when on settings
+* SeedVault no longer shows up in backup status
```

Comment:
I think we can exclude this, it confused me to see this mentioned before I found the commit and figured out what it means.